docs/usage/key-concepts/scheduling.md

```diff
@@ -93,17 +93,13 @@ Some config examples:
 
 ```json title="Renovate should run each day before 4 am"
 {
-  "schedule": ["before 4am"]
+  "schedule": ["* 0-3 * * *"]
 }
 ```
 
 ```json title="Renovate should run outside of common office hours"
 {
-  "schedule": [
-    "after 10pm every weekday",
-    "before 5am every weekday",
-    "every weekend"
-  ]
+  "schedule": ["* 22-23 * * 1-5", "* 0-4 * * 1-5", "* * * * 6,7"]
```

Comment:
```suggestion
  "schedule": ["* 22-24 * * 1-5", "* 0-4 * * 0-5", "* * * * 6,7"]
```
is this valid?
```suggestion
  "schedule": ["* 0-5,22-24 * * 1-5", "* * * * 6,7"]
```



do we need `24` or `0` for midnight?

Reply:
24 is not valid for croner and 22-23 is correct as it covers from 10pm to 11.59pm
And 0-5 covers from midnight to 5.59am